spec/request/documents_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe 'Documents management', type: :request do
+  it 'should fail if no file is provided' do
+    post '/v0/claims/1/documents'
+    expect(response).to_not be_success
+  end
+
+  it 'should upload a file' do
+    VCR.use_cassette('evss/documents/create') do
```

Comment:
can this use the same cassette as https://github.com/department-of-veterans-affairs/vets-api/pull/70/files#diff-3d8627f9748858ab61ddcafe9d3a29c2R36? it looks like the same request (just different claim_id but could change that in one of these specs)